cluster/engine.go

```diff
@@ -491,6 +492,14 @@ func (e *Engine) updateSpecs() error {
 		e.lastError = message
 		return fmt.Errorf(message)
 	}
+
+	if info.SystemTime != "" {
+		engineTime, _ := time.Parse(time.RFC3339Nano, info.SystemTime)
+		e.DeltaDuration = time.Now().UTC().Sub(engineTime)
```

Comment:
This delta is an estimation of time difference between manager and engine with adjustment of delays (Engine response delay + network delay + manager process delay). The adjustment of delays is not a constant. If the servers are sync up on time, this delta might be the source of error. 

Another way to fix is to output time difference as error in log and in `docker ps`, but without adjusting time. 

I think both approaches are ok.